judge/judgedaemon.main.php

```diff
@@ -1284,7 +1284,7 @@ private function compile(
 
             // Revoke readablity for domjudge-run user to this workdir.
             chmod($workdir, 0700);
-            logmsg(LOG_NOTICE, "Judging s$judgeTask[submitid], task $judgeTask[judgetaskid]: compile error");
+            logmsg(LOG_NOTICE, "Judging $judgeTask[submitid], task $judgeTask[judgetaskid]: compile error");
```

Comment:
Are you sure the `s` should be gone? There can be multiple judgings for 1 submitid right? So maybe this should be an `j` actually? Or do we now prefix the externalid with the type so `s`?